### ISSUE TITLE
Crash when opening

### ISSUE THREAD

**Issue by JosefNemec** [felixkmh/StartPage-for-Playnite#12]
I just installed it from add-on browser and clicked the button, crashes every time.

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at LandingPage.ViewModels.LandingPageViewModel.Update() in D:\Programmierung\playnite-dev\LandingPage\source\ViewModels\LandingPageViewModel.cs:line 73
   at LandingPage.LandingPageExtension.get_ViewModel() in D:\Programmierung\playnite-dev\LandingPage\source\LandingPageExtension.cs:line 67
   at LandingPage.LandingPageExtension.<GetSidebarItems>b__21_0() in D:\Programmierung\playnite-dev\LandingPage\source\LandingPageExtension.cs:line 90
   at Playnite.DesktopApp.ViewModels.SidebarWrapperItem.Activation(Object arg) in C:\projects\playnite\source\Playnite.DesktopApp\ViewModels\DesktopAppViewModel_Sidebar.cs:line 130
   at Playnite.SDK.RelayCommand`1.Execute(Object parameter) in C:\projects\playnite\source\PlayniteSDK\RelayCommand.cs:line 199
   at MS.Internal.Commands.CommandHelpers.CriticalExecuteCommandSource(ICommandSource commandSource, Boolean userInitiated)
```

**Comment by felixkmh** [comment]
It's because the SuccessStory ViewModel is not being created but it's trying to call a function on it. So I assume that it's not installed. Should be an easy fix.

**Comment by JosefNemec** [comment]
I don't have any 3rd party extensions installed.

**Comment by felixkmh** [comment]
fixed in v1.0.4
